Documentation/CRDs/Cluster/external-cluster/external-cluster.md

```diff
@@ -19,324 +19,26 @@ Create the desired types of storage in the provider Ceph cluster:
 * [RBD pools](https://docs.ceph.com/en/latest/rados/operations/pools/#create-a-pool)
 * [CephFS filesystem](https://docs.ceph.com/en/quincy/cephfs/createfs/)
 
-## Commands on the source Ceph cluster
+## Steps to connect external Provider cluster to Consumer cluster
 
-In order to configure an external Ceph cluster with Rook, we need to extract some information in order to connect to that cluster.
+* [Extract data from Provider Ceph cluster](/Documentation/CRDs/Cluster/external-cluster/provider-cluster.md)
 
-### 1. Create all users and keys
+* [Commands on the K8s consumer cluster](/Documentation/CRDs/Cluster/external-cluster/consumer-cluster.md)
 
-Run the python script [create-external-cluster-resources.py](https://github.com/rook/rook/blob/master/deploy/examples/external/create-external-cluster-resources.py) for creating all users and keys.
+* [Verify the cluster connection](/Documentation/CRDs/Cluster/external-cluster/consumer-cluster.md#cluster-verification)
```

Comment:
This is part of the import in the previous step, so perhaps we don't need this step. It can be mentioned in the previously bullet.